src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -86,10 +106,14 @@ const SystemsTable = ({ systemsFetchStatus, fetchSystems, systems, intl, filters
     }, [filters, history]);
 
     useEffect(() => {
-        if (!filterBuilding && filters.sort !== undefined) {
```

Comment:
need filterbuilding here, otherwise on page refresh you'll pound the api 4 times (we read sort params from the url if they're present and set them, if not present we'll push the default which is updated)

Reply:
When I left this on the initial filter was not applied. Meaning when you refreshed page the URL was changed, but table was not sorted. I'll give that a try and see if it'll actually fire multiple requests.